deploy/gatekeeper.yaml

```diff
@@ -157,6 +264,41 @@ rules:
   - patch
   - delete
 ---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  creationTimestamp: null
+  name: gatekeeper-manager-rolebinding
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: gatekeeper-manager-role
+subjects:
+- kind: ServiceAccount
+  name: default
+  namespace: system
```

Comment:
`make deploy`/kustomize updates the namespace value to `gatekeeper-system`

Reply:
Good catch.

For me, make deploy is leaving the namespace as system. I think this may have been a change in Kustomize behavior. I've amended the config so it writes gatekeeper-system.

Can you verify that when you run `make manifests` with the latest version of this change gatekeeper.yaml is unchanged for you?

Reply:
Running `make manifests` updates rbac_role_binding.yaml to `namespace: system` and deploy/gatekeeper.yaml to `namespace: system` namespace: gatekeeper-system`

Reply:
so the namespace correctly points to `gatekeeper-system` for you in gatekeeper.yaml?

Reply:
This is what I get for running kustomize from HEAD.

Should be fixed now.